source/Cosmos.IL2CPU/CosmosAssembler.cs

```diff
@@ -43,18 +42,19 @@ public virtual void WriteDebugVideo(string aText)
             var xPreBootLogging = RemoveBootDebugOutput;
             if (xPreBootLogging)
             {
-                new Comment("DebugVideo '" + aText + "'");
+                _ = new Comment("DebugVideo '" + aText + "'");
```

Comment:
I would prefer if we dont have the _ = everywhere

Reply:
> I would prefer if we dont have the _ = everywhere

It's the normal .net way, do your just want me to remove it? It will cause more warnings

Reply:
Please disable the warnings if possible